ui/pages/settings/advanced-tab/advanced-tab.component.js

```diff
@@ -238,13 +243,73 @@ export default class AdvancedTab extends PureComponent {
     );
   }
 
+  renderToggleStxOptIn() {
+    const { t } = this.context;
+    const { stxOptIn, setStxOptIn } = this.props;
+
+    const learMoreLink = (
+      <ButtonLink
+        size={ButtonLinkSize.Inherit}
+        textProps={{
+          variant: TextVariant.bodyMd,
+          alignItems: AlignItems.flexStart,
+        }}
+        as="a"
+        href="https://support.metamask.io/hc/en-us/articles/9184393821211"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        {t('learnMoreUpperCase')}
+      </ButtonLink>
+    );
+
+    return (
+      <Box
```

Comment:
I mention this everytime I see a PR with a new toggle, but this file would welcome an `AdvancedToggleButton` local function to drastically reduce the total size and duplication.

Could even be more general and used in the other settings components such as privacy.

Not needed now, but just for audit sake.